File: SDKs/Aspose.Words_Cloud_SDK_for_CSharp/src/Com/Aspose/Words/Model/DocumentEntryList.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Words.Model {
  public class DocumentEntryList {
    public List<DocumentEntry> DocumentEntries { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class DocumentEntryList {\n");
      sb.Append("  DocumentEntries: ").Append(DocumentEntries).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Words-Cloud-SDK-for-Java/src/main/java/com/aspose/words/model/DocumentEntryList.java
package com.aspose.words.model;

import java.util.*;
public class DocumentEntryList {
  private List<DocumentEntry> DocumentEntries = new ArrayList<DocumentEntry>();
  /**
	 * getDocumentEntries
	 * Gets List<DocumentEntry>
	 * @return DocumentEntries
	 */
  public List<DocumentEntry> getDocumentEntries() {
    return DocumentEntries;
  }

	/**
	 * setDocumentEntries
	 * Sets List<DocumentEntry>
	 * @param DocumentEntries List<DocumentEntry>
	 */
  public void setDocumentEntries(List<DocumentEntry> DocumentEntries) {
    this.DocumentEntries = DocumentEntries;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class DocumentEntryList {\n");
    sb.append("  DocumentEntries: ").append(DocumentEntries).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

